cs4660/graph/graph.py

```diff
@@ -86,22 +117,81 @@ def __init__(self):
         self.adjacency_list = {}
 
     def adjacent(self, node_1, node_2):
-        pass
+        # if node_1 is in the adjacency_list
+        if node_1 in self.adjacency_list:
+            # for every edge in the list of edges that belongs to node_1 which is adjacency_list[node_1]
+            for currentEdge in self.adjacency_list[node_1]:
+                #if node_2 is equal to the to_node in the current edge return true
+                if(node_2==currentEdge.to_node):
+                    return True
+            return False
```

Comment:
Probably also need to handle the case that `node_1` not exists in the self.adjacency_list case. That case, you should return false.